includes/specials/SpecialDeletedContributions.php

```diff
@@ -365,13 +386,13 @@ function getSubTitle( $userObj ) {
 							array(),
 							array(
 								'action' => 'unblock',
-								'ip' => $nt->getDBkey()
+								'ip' => $userTitle->getDBkey()
 							)
 						);
 					}
 					else { # User is not blocked
 						$tools[] = Linker::linkKnown( # Block link
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)